src/pybind/mgr/dashboard/frontend/e2e/rgw/buckets.po.ts

```diff
@@ -1,5 +1,157 @@
+import { $, browser, by, element } from 'protractor';
+// import { protractor } from 'protractor/built/ptor';
+import { Helper } from '../helper.po';
 import { PageHelper } from '../page-helper.po';
 
-export class BucketsPage extends PageHelper {
-  pages = { index: '/#/rgw/bucket' };
+export class BucketsPageHelper extends PageHelper {
+  static pages = {
+    index: '/#/rgw/bucket',
+    create: '/#/rgw/bucket/create'
+  };
+
+  static navigateTo(page = null) {
+    return browser.get(this.pages[page || 'index']);
+  }
+
+  static create(name, owner) {
+    this.navigateTo('create');
+    expect(PageHelper.getTitleText()).toBe('Create Bucket');
+
+    // Need the string '[value="<owner>"]' to find the owner in dropdown menu
+    const str1 = '[value="';
+    const str2 = '"]';
+    const str3 = str1.concat(owner);
+    const getOwnerStr = str3.concat(str2);
+
+    // Enter in bucket name
+    element(by.id('bid')).click();
+    element(by.id('bid')).sendKeys(name);
+
+    // Select bucket owner
+    element(by.id('owner')).click();
+    $(getOwnerStr).click();
+    expect(element(by.id('owner')).getAttribute('class')).toContain('ng-valid');
+
+    // Click the create button and wait for bucket to be made
+    element(by.cssContainingText('button', 'Create Bucket'))
+      .click()
+      .then(() => {
+        browser.sleep(5000);
+        this.navigateTo();
+        browser.wait(Helper.EC.presenceOf(PageHelper.getTableCell(name)), 10000);
+      });
+  }
+
+  static edit(name, new_owner) {
+    const base_url = '/#/rgw/bucket/edit/';
+    const edit_url = base_url.concat(name);
+    browser.get(edit_url);
+
+    // Need the string '[value="<owner>"]' to find the new owner in dropdown menu
+    const str1 = '[value="';
+    const str2 = '"]';
+    const str3 = str1.concat(new_owner);
+    const getOwnerStr = str3.concat(str2);
```

Comment:
You only need 1 const to create the desired string by using [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals):
```
const getOwnerStr = `[value="${new_owner}"]`;
```